roles/v2plugin/defaults/main.yml

```diff
@@ -0,0 +1,37 @@
+---
+
+# Role defaults for v2plugin
+# NOTE: Role defaults have a lower priority than inventory vars.
+# Include variables which need to be overridden by inventory vars here.
+
+contiv_v2plugin_image: "contiv/v2plugin:0.1"
+
+contiv_network_mode: "standalone" # Accepted values: standalone, aci
+netplugin_mode: "docker" # Accepted values: docker, kubernetes
+fwd_mode: "bridge" #Accepted values: bridge , routing
+
+cluster_store: "etcd://127.0.0.1:2379"
+contiv_standalone_binaries: "/var/contiv_cache"
+ofnet_master_port: 9001
+ofnet_agent_port1: 9002
+ofnet_agent_port2: 9003
+netmaster_port: 9999
+gobgp_grpc_port: 8080
+bgp_port: 179
+# Since there is a conflict on vxlan port used by Docker overlay (ingress)
+# contiv needs to use a different port than the standard 4789
+vxlan_port: 8472
```

Comment:
Specific reason to choose this port number? Why not stick with the same port as what contiv_network chooses? Just that it is easier to debug in different modes if the same port is chosen.